restaurants/padowetz

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python
+# vim: set ts=2 sw=2 et fileencoding=utf-8:
```

Comment:
indentation should be multiple of 4 - PEP8 FTW

Reply:
I don't care.